react-components/src/components/CadModelContainer/CadModelContainer.tsx

```diff
@@ -44,7 +44,7 @@ export function CadModelContainer({
     addModel(addModelOptions, transform)
       .then((model) => {
         onLoad?.(model);
-        setRevealResourcesCount(viewer.models.length);
+        setRevealResourcesCount(viewer.models.length + viewer.get360ImageCollections().length);
```

Comment:
I would prefer to use a const variable to set this summed counter, but it is more a personal taste, TBH. We can keep this as it is, if you prefer :)

Reply:
I agree that this should be more standardized across different model types... I added a `getViewerResourceCount` function that is used on all locations